tests/verifyPMMInventory_test.js

```diff
@@ -175,3 +175,41 @@ Scenario(
     }
   },
 );
+
+Scenario(
+  'PMM-T1226 - Verify Agents has process_exec_path option on Inventory page @inventory @exporters @nightly',
+  async ({ I, pmmInventoryPage }) => {
+    I.amOnPage(pmmInventoryPage.url);
+    await I.waitForVisible(pmmInventoryPage.fields.agentsLink, 20);
+    I.click(pmmInventoryPage.fields.agentsLink);
+    await I.waitForVisible(pmmInventoryPage.fields.tableRow);
+    const agentTextValues = await I.grabTextFromAll(pmmInventoryPage.fields.processExecPathExporters);
+
+    agentTextValues.forEach((value) => {
+      if (!value.toLowerCase().includes('qan')) {
+        assert.ok(value.includes('process_exec_path'), `process_exec_path is not present for exporter ${value}`);
+        const newValue = value.replace('process_exec_path:', '').trim();
+
+        assert.ok(newValue.length > 0, `process_exec_path value is empty for ${value}`);
+      }
+    });
+  },
+);
+
+Scenario(
+  'PMM-T1225 - Verify summary file includes process_exec_path for agents @inventory @exporters @nightly',
```

Comment:
also, this test won't work on nightly. @puneet0191 where these tests should be executed?

Reply:
why it won't work on Nightly?, I think it is fine to run them on Nightly, we have all exporters on Nightly

Reply:
as far as I remember we don't have `pmm-admin` on a worker. right?

Reply:
you right, this should be part of matrix, add it to CLI or pmm-pgsm integration stage